.github/workflows/ci.yml

```diff
@@ -16,15 +16,15 @@ jobs:
         python-version: ['3.8', '3.9', '3.10', '3.11']
     name: Tests - Python ${{ matrix.python-version }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           path: buildstockbatch
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
Any particular reason to switch to v4 or just updating?

Reply:
There are a ton of deprecation annotations on the Actions tab about these...